tests/TestRunner/tmp_cluster.py

```diff
@@ -37,9 +37,23 @@ def __exit__(self, xc_type, exc_value, traceback):
 
 
 if __name__ == '__main__':
-    parser = ArgumentParser()
+    parser = ArgumentParser(formatter_class=RawDescriptionHelpFormatter,
+                            description="""
+    This script automatically configures a temporary local cluster on the machine
+    and then calls a command while this cluster is running. As soon as the command
+    returns, the configured cluster is killed and all generated data is deleted.
+    This is useful for testing: if a test needs access to a fresh fdb cluster, one
+    can simply pass the test command to this script.
+
+    The command to run after the cluster started. Before the command is execute,
```

Comment:
executed* :)

Reply:
Done. BTW: You can make suggestions in comments in GitHub ;) (just press CMD-G):
```suggestion
    The command to run after the cluster started. Before the command is executed,
```